frontend/src/pages/premium.page.tsx

```diff
@@ -236,7 +236,14 @@ const PremiumPromo: NextPage = () => {
 
   const getPerkCta = (label: keyof typeof perkCtaRefs) => {
     if (!isPremiumAvailableInCountry(runtimeData.data)) {
-      return null;
+      return (
+        <LinkButton
+          href="#pricing"
```

Comment:
I think this should point to the waitlist?

```suggestion
          href="/premium/waitlist"
```